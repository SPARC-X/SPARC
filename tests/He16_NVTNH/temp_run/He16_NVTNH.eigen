Final eigenvalues and occupation numbers

kred #1 = (0.000000,0.000000,0.000000)
n        eigval                 occ
1       -6.543088648056E-01     2.000000000000
2       -5.584191658667E-01     2.000000000000
3       -5.550566531264E-01     2.000000000000
4       -4.076222634143E-01     2.000000000000
5       -4.071054146073E-01     2.000000000000
6       -4.054685672980E-01     2.000000000000
7       -3.319271606357E-01     2.000000000000
8       -3.287322015004E-01     2.000000000000
9       -3.264407497222E-01     2.000000000000
10      -3.231730201933E-01     2.000000000000
11      -2.230489038475E-01     2.000000000000
12      -2.207470903659E-01     2.000000000000
13      -2.198177520067E-01     2.000000000000
14      -1.595102096965E-01     2.000000000000
15      -1.553429709736E-01     2.000000000000
16      -8.063866943042E-02     2.000000000000
17       2.900288421034E-01     0.000000042692
18       2.903111166447E-01     0.000000038191
19       2.946096969749E-01     0.000000007000
20       4.190553554629E-01     0.000000000000
21       4.199443721647E-01     0.000000000000
22       4.202962883554E-01     0.000000000000
23       4.236894247739E-01     0.000000000000
24       6.291848877439E-01     0.000000000000
