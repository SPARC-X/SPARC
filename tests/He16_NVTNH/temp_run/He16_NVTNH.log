
error = 3.69e-08 > TOL_PSEUDOCHARGE = 1.00e-08

WARNING: upperbond for pseudocharge radius (Rbmax_?) is not big enough! Rbmax_x = 6.3, Rbmax_y = 6.3, Rbmax_z = 6.3

WARNING: SCF#4 did not converge to desired accuracy!
WARNING: SCF#5 did not converge to desired accuracy!
The program took 178.641 s.
