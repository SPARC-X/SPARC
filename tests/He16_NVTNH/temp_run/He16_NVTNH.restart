:MDSTEP: 10
:R(Bohr):
  1.1852244038E-02   2.1696830635E-03   1.9265556781E-02
  3.3840631009E-02   5.1589327278E-02   3.0955093200E+00
  6.6452186921E-02   3.0945233301E+00   2.0927909427E-03
  1.1634772976E-01   3.0790149815E+00   3.0644342061E+00
  3.1550227348E+00   4.6036926671E-02   2.3181798087E-02
  3.0569297888E+00   4.5149429945E-02   3.0842874483E+00
  3.1557158642E+00   3.0616080190E+00   4.8312816043E-02
  3.1540844761E+00   3.0776019733E+00   3.0491315963E+00
  6.4136564151E+00   6.0719673717E+00   6.0440928708E+00
  6.1505657222E+00   1.6327987664E-01   2.8765061711E+00
  6.0689097289E+00   3.3065648499E+00   8.5943682094E-02
  6.1665355837E+00   3.1916975792E+00   3.2593934143E+00
  9.1670674938E+00   1.7896327164E-02   5.9544641785E+00
  8.9697600731E+00   1.2748150165E-02   3.0581006677E+00
  9.3464245283E+00   2.9494293522E+00   6.0448339778E+00
  9.2345230332E+00   3.0360733918E+00   2.8951523214E+00
:V(Bohr/atu):
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  1.2419829683E-03   1.4356855098E-04  -5.0373731326E-04
 -4.0477698883E-04   6.5481021293E-04  -9.6994693161E-04
 -1.0109412926E-04   9.8255090553E-04   4.7645230576E-04
  3.0551045452E-04   5.9787811361E-04   9.2929557843E-04
 -1.0251509701E-04  -2.5990534142E-05  -6.6229075059E-04
 -1.0575568278E-03  -3.9711891057E-06  -1.3193812784E-05
  3.9179171656E-04  -6.1320249454E-04  -4.2250566460E-04
  2.1171533805E-04  -1.6827706201E-04  -8.2241265308E-04
:snose: 0.0178485001045671
:xinose: 0.000223963544348678
:TTHRMI(K): 800
:TEL(K): 800
:TIO(K): 853.750909255247
