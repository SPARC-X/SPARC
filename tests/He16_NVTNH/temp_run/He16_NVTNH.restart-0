:MDSTEP: 9
:R(Bohr):
  1.1852244038E-02   2.1696830635E-03   1.9265556781E-02
  3.3840631009E-02   5.1589327278E-02   3.0955093200E+00
  6.6452186921E-02   3.0945233301E+00   2.0927909427E-03
  1.1634772976E-01   3.0790149815E+00   3.0644342061E+00
  3.1550227348E+00   4.6036926671E-02   2.3181798087E-02
  3.0569297888E+00   4.5149429945E-02   3.0842874483E+00
  3.1557158642E+00   3.0616080190E+00   4.8312816043E-02
  3.1540844761E+00   3.0776019733E+00   3.0491315963E+00
  6.3826483753E+00   6.0692419463E+00   6.0556679428E+00
  6.1593795581E+00   1.4748142297E-01   2.9000350498E+00
  6.0712644463E+00   3.2813887006E+00   7.5213973513E-02
  6.1594015394E+00   3.1767601017E+00   3.2368541911E+00
  9.1705072428E+00   1.8057020681E-02   5.9713096580E+00
  8.9960919850E+00   1.2650195570E-02   3.0583149465E+00
  9.3362626146E+00   2.9647504138E+00   6.0548982821E+00
  9.2294011922E+00   3.0404483437E+00   2.9154968562E+00
:V(Bohr/atu):
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  1.2553190103E-03   7.8910199478E-05  -4.3339110357E-04
 -3.1059432340E-04   6.2021879122E-04  -9.2975121859E-04
 -9.1163504506E-05   1.0450515379E-03   3.9343403202E-04
  2.7106797884E-04   6.0593608601E-04   8.9001359656E-04
 -1.6788142400E-04   1.0060493800E-05  -6.9335891792E-04
 -1.0652603618E-03   1.0701881141E-05  -4.2378237966E-06
  4.2618972013E-04  -6.2097829308E-04  -3.9085563063E-04
  2.0194496463E-04  -1.8291232229E-04  -8.1842236536E-04
:snose: 0.0130531292291342
:xinose: 0.000172633927347226
:TTHRMI(K): 800
:TEL(K): 800
:TIO(K): 836.296930088179
