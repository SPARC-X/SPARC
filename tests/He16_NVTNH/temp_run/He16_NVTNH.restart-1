:MDSTEP: 6
:R(Bohr):
  1.1852244038E-02   2.1696830635E-03   1.9265556781E-02
  3.3840631009E-02   5.1589327278E-02   3.0955093200E+00
  6.6452186921E-02   3.0945233301E+00   2.0927909427E-03
  1.1634772976E-01   3.0790149815E+00   3.0644342061E+00
  3.1550227348E+00   4.6036926671E-02   2.3181798087E-02
  3.0569297888E+00   4.5149429945E-02   3.0842874483E+00
  3.1557158642E+00   3.0616080190E+00   4.8312816043E-02
  3.1540844761E+00   3.0776019733E+00   3.0491315963E+00
  6.2885538864E+00   6.0695873140E+00   5.7223314615E-03
  6.1735569632E+00   1.0472371690E-01   2.9655421696E+00
  6.0776846039E+00   3.1974734843E+00   5.3707622269E-02
  6.1426598044E+00   3.1309847433E+00   3.1743569509E+00
  9.1882051626E+00   1.4223595306E-02   6.0255953023E+00
  9.0760658158E+00   1.0619893119E-02   3.0577246223E+00
  9.3010941416E+00   3.0114561830E+00   5.4880779273E-03
  9.2152296546E+00   3.0551908700E+00   2.9761231383E+00
:V(Bohr/atu):
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  0.0000000000E+00   0.0000000000E+00   0.0000000000E+00
  1.2651584328E-03  -7.3993025943E-05  -2.6943409423E-04
 -9.0530774025E-05   5.3577909924E-04  -8.4013629711E-04
 -8.9980113704E-05   1.1988171631E-03   2.0233436386E-04
  1.8351352537E-04   6.2110731561E-04   7.9769293274E-04
 -2.8445888242E-04   8.1587880821E-05  -7.5674579169E-04
 -1.0811766895E-03   3.8512384050E-05   1.8234291535E-05
  5.1500351364E-04  -6.2958617247E-04  -3.1893573520E-04
  1.8199270902E-04  -2.0707816017E-04  -8.1322234609E-04
:snose: 0.00389627401916492
:xinose: 8.37280383940852e-05
:TTHRMI(K): 800
:TEL(K): 800
:TIO(K): 821.122300960514
