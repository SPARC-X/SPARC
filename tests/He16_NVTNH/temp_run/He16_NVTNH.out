***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 21:01:59 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 12.151112185593 6.0755560927965 6.0755560927965 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 152 76 76
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
ELEC_TEMP: 800
CHEB_DEGREE: 54
NSTATES: 24
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 1
TWTIME: 1E+09
MD_FLAG: 1
MD_METHOD: NVT_NH
MD_TIMESTEP: 0.6
MD_NSTEP: 10
ION_VEL_DSTR: 1
ION_VEL_DSTR_RAND: 0
ION_TEMP: 800
ION_TEMP_END: 800
QMASS: 1653.65493345972
RESTART_FLAG: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 10
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.8
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 1
PRINT_DENSITY: 0
PRINT_MDOUT: 1
PRINT_VELS: 1
PRINT_RESTART: 1
PRINT_RESTART_FQ: 3
OUTPUT_FILE: He16_NVTNH
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 24
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 4 2 3
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing                       :  0.0799415 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  He16_NVTNH.out
Final eigenvalues printed to       :  He16_NVTNH.eigen
MD output printed to               :  He16_NVTNH.aimd
Total number of atom types         :  1
Total number of atoms              :  16
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  He 2
Pseudopotential                    :  He-2-0.5-loc.psp8
Atomic mass                        :  4.002602
Pseudocharge radii of atom type 1  :  6.32 6.32 6.32 (x, y, z dir)
Number of atoms of type 1          :  16
Estimated total memory usage       :  1.13 GB
Estimated memory per processor     :  48.00 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7475605576E+00        3.268E-02        6.013
2            -2.7775519956E+00        3.043E-02        2.190
3            -2.7786659394E+00        1.470E-02        2.001
4            -2.7787327075E+00        2.992E-03        1.970
5            -2.7787347994E+00        2.651E-04        1.897
6            -2.7787348765E+00        3.794E-05        1.694
7            -2.7787348648E+00        1.517E-05        1.492
8            -2.7787348678E+00        2.267E-06        1.437
9            -2.7787348803E+00        3.841E-07        1.319
Total number of SCF: 9     

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787348803E+00 (Ha/atom)
Total free energy                  : -4.4459758084E+01 (Ha)
Band structure energy              : -1.0687617768E+01 (Ha)
Exchange correlation energy        : -1.6848479331E+01 (Ha)
Self and correction energy         : -2.9009607443E+02 (Ha)
Entropy*kb*T                       : -4.2394198369E-09 (Ha)
Fermi level                        :  2.4257471960E-01 (Ha)
RMS force                          :  2.9281833618E-03 (Ha/Bohr)
Maximum force                      :  9.6411868354E-03 (Ha/Bohr)
Time for force calculation         :  0.285 (sec)
Pressure                           :  1.3438241398E+02 (GPa)
Maximum stress                     :  1.6824938822E+02 (GPa)
Time for stress calculation        :  0.820 (sec)
MD step time                       :  22.284 (sec)
===================================================================
                    Self Consistent Field (SCF#2)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7786359348E+00        1.563E-02        2.243
2            -2.7787522347E+00        3.370E-03        2.025
3            -2.7787547137E+00        2.477E-04        1.899
4            -2.7787548013E+00        6.418E-05        1.720
5            -2.7787548019E+00        1.327E-05        1.643
6            -2.7787548027E+00        3.725E-06        1.461
7            -2.7787548035E+00        1.198E-06        1.370
8            -2.7787548027E+00        2.325E-07        1.336
Total number of SCF: 8     

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787548027E+00 (Ha/atom)
Total free energy                  : -4.4460076844E+01 (Ha)
Band structure energy              : -1.0688047138E+01 (Ha)
Exchange correlation energy        : -1.6848164023E+01 (Ha)
Self and correction energy         : -2.9009450281E+02 (Ha)
Entropy*kb*T                       : -4.2365750472E-09 (Ha)
Fermi level                        :  2.4263147005E-01 (Ha)
RMS force                          :  2.8129240866E-03 (Ha/Bohr)
Maximum force                      :  7.8444217236E-03 (Ha/Bohr)
Time for force calculation         :  0.284 (sec)
Pressure                           :  1.3433920628E+02 (GPa)
Maximum stress                     :  1.6823756333E+02 (GPa)
Time for stress calculation        :  0.817 (sec)
MD step time                       :  15.959 (sec)
===================================================================
                    Self Consistent Field (SCF#3)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7786533399E+00        1.590E-02        2.246
2            -2.7787712677E+00        3.449E-03        2.034
3            -2.7787738070E+00        2.443E-04        1.891
4            -2.7787738961E+00        6.526E-05        1.724
5            -2.7787738958E+00        1.368E-05        1.659
6            -2.7787738966E+00        3.881E-06        1.453
7            -2.7787738983E+00        1.227E-06        1.371
8            -2.7787738972E+00        2.377E-07        1.337
Total number of SCF: 8     

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787738972E+00 (Ha/atom)
Total free energy                  : -4.4460382356E+01 (Ha)
Band structure energy              : -1.0689013525E+01 (Ha)
Exchange correlation energy        : -1.6847851024E+01 (Ha)
Self and correction energy         : -2.9009266799E+02 (Ha)
Entropy*kb*T                       : -4.2384808426E-09 (Ha)
Fermi level                        :  2.4276516805E-01 (Ha)
RMS force                          :  2.9754830589E-03 (Ha/Bohr)
Maximum force                      :  9.7647155747E-03 (Ha/Bohr)
Time for force calculation         :  0.284 (sec)
Pressure                           :  1.3428922856E+02 (GPa)
Maximum stress                     :  1.6822598462E+02 (GPa)
Time for stress calculation        :  0.823 (sec)
MD step time                       :  15.996 (sec)
===================================================================
                    Self Consistent Field (SCF#4)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7787499698E+00        2.869E-04        1.856
2            -2.7787807818E+00        3.289E-04        1.787
3            -2.7787814376E+00        1.330E-04        1.764
4            -2.7787814487E+00        1.054E-04        1.602
5            -2.7787814496E+00        6.845E-05        1.568
6            -2.7787814537E+00        5.731E-05        1.390
7            -2.7787814487E+00        1.506E-05        1.562
8            -2.7787814533E+00        1.030E-05        1.380
9            -2.7787814549E+00        4.586E-06        1.388
10           -2.7787814524E+00        1.100E-06        1.365
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
WARNING: SCF#4 did not converge to desired accuracy!
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787814524E+00 (Ha/atom)
Total free energy                  : -4.4460503239E+01 (Ha)
Band structure energy              : -1.0690437389E+01 (Ha)
Exchange correlation energy        : -1.6847504588E+01 (Ha)
Self and correction energy         : -2.9009140095E+02 (Ha)
Entropy*kb*T                       : -4.2365781083E-09 (Ha)
Fermi level                        :  2.4296701023E-01 (Ha)
RMS force                          :  3.3877333805E-03 (Ha/Bohr)
Maximum force                      :  1.2404206328E-02 (Ha/Bohr)
Time for force calculation         :  0.283 (sec)
Pressure                           :  1.3423215923E+02 (GPa)
Maximum stress                     :  1.6821303071E+02 (GPa)
Time for stress calculation        :  0.816 (sec)
MD step time                       :  17.846 (sec)
===================================================================
                    Self Consistent Field (SCF#5)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7787610381E+00        2.768E-04        1.873
2            -2.7787918213E+00        3.219E-04        1.784
3            -2.7787924690E+00        1.337E-04        1.763
4            -2.7787924791E+00        1.056E-04        1.606
5            -2.7787924876E+00        5.859E-05        1.590
6            -2.7787924842E+00        4.912E-05        1.383
7            -2.7787924842E+00        9.447E-06        1.496
8            -2.7787924845E+00        6.598E-06        1.367
9            -2.7787924828E+00        5.779E-06        1.347
10           -2.7787924844E+00        1.160E-06        1.386
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
WARNING: SCF#5 did not converge to desired accuracy!
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787924844E+00 (Ha/atom)
Total free energy                  : -4.4460679750E+01 (Ha)
Band structure energy              : -1.0692508761E+01 (Ha)
Exchange correlation energy        : -1.6847130658E+01 (Ha)
Self and correction energy         : -2.9009266598E+02 (Ha)
Entropy*kb*T                       : -4.2344458480E-09 (Ha)
Fermi level                        :  2.4323605715E-01 (Ha)
RMS force                          :  4.2950161676E-03 (Ha/Bohr)
Maximum force                      :  1.5481324839E-02 (Ha/Bohr)
Time for force calculation         :  0.285 (sec)
Pressure                           :  1.3416259572E+02 (GPa)
Maximum stress                     :  1.6819576732E+02 (GPa)
Time for stress calculation        :  0.818 (sec)
MD step time                       :  17.780 (sec)
===================================================================
                    Self Consistent Field (SCF#6)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7787701895E+00        2.818E-04        1.866
2            -2.7788008996E+00        3.144E-04        1.787
3            -2.7788015399E+00        1.330E-04        1.750
4            -2.7788015508E+00        1.036E-04        1.601
5            -2.7788015525E+00        6.586E-05        1.573
6            -2.7788015557E+00        5.546E-05        1.425
7            -2.7788015568E+00        1.069E-05        1.576
8            -2.7788015561E+00        8.420E-06        1.336
9            -2.7788015578E+00        1.687E-06        1.398
10           -2.7788015548E+00        6.124E-07        1.309
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7788015548E+00 (Ha/atom)
Total free energy                  : -4.4460824878E+01 (Ha)
Band structure energy              : -1.0695217651E+01 (Ha)
Exchange correlation energy        : -1.6846657374E+01 (Ha)
Self and correction energy         : -2.9009239449E+02 (Ha)
Entropy*kb*T                       : -4.2262407458E-09 (Ha)
Fermi level                        :  2.4356208525E-01 (Ha)
RMS force                          :  5.4440458098E-03 (Ha/Bohr)
Maximum force                      :  1.9428491950E-02 (Ha/Bohr)
Time for force calculation         :  0.283 (sec)
Pressure                           :  1.3407637721E+02 (GPa)
Maximum stress                     :  1.6817146505E+02 (GPa)
Time for stress calculation        :  0.817 (sec)
MD step time                       :  17.813 (sec)
===================================================================
                    Self Consistent Field (SCF#7)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7787808095E+00        2.730E-04        1.869
2            -2.7788114225E+00        3.043E-04        1.779
3            -2.7788120477E+00        1.322E-04        1.759
4            -2.7788120584E+00        1.034E-04        1.606
5            -2.7788120669E+00        6.083E-05        1.594
6            -2.7788120668E+00        4.940E-05        1.410
7            -2.7788120613E+00        9.971E-06        1.569
8            -2.7788120625E+00        7.229E-06        1.346
9            -2.7788120631E+00        3.357E-06        1.382
10           -2.7788120597E+00        8.005E-07        1.353
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7788120597E+00 (Ha/atom)
Total free energy                  : -4.4460992954E+01 (Ha)
Band structure energy              : -1.0698650862E+01 (Ha)
Exchange correlation energy        : -1.6846019595E+01 (Ha)
Self and correction energy         : -2.9009196382E+02 (Ha)
Entropy*kb*T                       : -4.2236318663E-09 (Ha)
Fermi level                        :  2.4394409507E-01 (Ha)
RMS force                          :  6.6731237726E-03 (Ha/Bohr)
Maximum force                      :  2.3406699874E-02 (Ha/Bohr)
Time for force calculation         :  0.282 (sec)
Pressure                           :  1.3396489163E+02 (GPa)
Maximum stress                     :  1.6813355121E+02 (GPa)
Time for stress calculation        :  0.816 (sec)
MD step time                       :  17.853 (sec)
===================================================================
                    Self Consistent Field (SCF#8)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7788044883E+00        2.687E-04        1.888
2            -2.7788350471E+00        2.925E-04        1.778
3            -2.7788356655E+00        1.311E-04        1.752
4            -2.7788356744E+00        1.022E-04        1.613
5            -2.7788356833E+00        6.058E-05        1.582
6            -2.7788356838E+00        5.052E-05        1.392
7            -2.7788356789E+00        1.136E-05        1.571
8            -2.7788356785E+00        1.018E-05        1.327
9            -2.7788356837E+00        2.884E-06        1.403
10           -2.7788356779E+00        4.376E-07        1.350
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7788356779E+00 (Ha/atom)
Total free energy                  : -4.4461370847E+01 (Ha)
Band structure energy              : -1.0702987580E+01 (Ha)
Exchange correlation energy        : -1.6845154065E+01 (Ha)
Self and correction energy         : -2.9009280687E+02 (Ha)
Entropy*kb*T                       : -4.2146238858E-09 (Ha)
Fermi level                        :  2.4436418135E-01 (Ha)
RMS force                          :  7.9277890743E-03 (Ha/Bohr)
Maximum force                      :  2.7363882110E-02 (Ha/Bohr)
Time for force calculation         :  0.282 (sec)
Pressure                           :  1.3381665059E+02 (GPa)
Maximum stress                     :  1.6807175252E+02 (GPa)
Time for stress calculation        :  0.826 (sec)
MD step time                       :  17.847 (sec)
===================================================================
                    Self Consistent Field (SCF#9)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7788385588E+00        2.685E-04        1.870
2            -2.7788692022E+00        2.816E-04        1.777
3            -2.7788698079E+00        1.302E-04        1.744
4            -2.7788698157E+00        9.974E-05        1.626
5            -2.7788698156E+00        6.587E-05        1.560
6            -2.7788698240E+00        5.267E-05        1.450
7            -2.7788698272E+00        8.748E-06        1.595
8            -2.7788698211E+00        7.248E-06        1.328
9            -2.7788698234E+00        2.749E-06        1.379
10           -2.7788698182E+00        2.980E-07        1.336
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7788698182E+00 (Ha/atom)
Total free energy                  : -4.4461917092E+01 (Ha)
Band structure energy              : -1.0708265075E+01 (Ha)
Exchange correlation energy        : -1.6843941817E+01 (Ha)
Self and correction energy         : -2.9009247695E+02 (Ha)
Entropy*kb*T                       : -4.2063701112E-09 (Ha)
Fermi level                        :  2.4481410176E-01 (Ha)
RMS force                          :  9.1846530108E-03 (Ha/Bohr)
Maximum force                      :  3.1231543952E-02 (Ha/Bohr)
Time for force calculation         :  0.301 (sec)
Pressure                           :  1.3362125072E+02 (GPa)
Maximum stress                     :  1.6798112788E+02 (GPa)
Time for stress calculation        :  0.814 (sec)
MD step time                       :  17.868 (sec)
===================================================================
                    Self Consistent Field (SCF#10)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7788999393E+00        2.619E-04        1.871
2            -2.7789309074E+00        2.708E-04        1.786
3            -2.7789315082E+00        1.290E-04        1.749
4            -2.7789315115E+00        1.010E-04        1.596
5            -2.7789315210E+00        5.936E-05        1.576
6            -2.7789315227E+00        4.994E-05        1.387
7            -2.7789315205E+00        1.124E-05        1.577
8            -2.7789315193E+00        9.831E-06        1.317
9            -2.7789315261E+00        3.322E-06        1.396
10           -2.7789315195E+00        3.785E-07        1.352
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7789315195E+00 (Ha/atom)
Total free energy                  : -4.4462904312E+01 (Ha)
Band structure energy              : -1.0714719289E+01 (Ha)
Exchange correlation energy        : -1.6842301939E+01 (Ha)
Self and correction energy         : -2.9009175830E+02 (Ha)
Entropy*kb*T                       : -4.1979609417E-09 (Ha)
Fermi level                        :  2.4528205429E-01 (Ha)
RMS force                          :  1.0425112326E-02 (Ha/Bohr)
Maximum force                      :  3.4935144606E-02 (Ha/Bohr)
Time for force calculation         :  0.282 (sec)
Pressure                           :  1.3336274810E+02 (GPa)
Maximum stress                     :  1.6785084292E+02 (GPa)
Time for stress calculation        :  0.812 (sec)
MD step time                       :  17.785 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  179.113 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
